cljfmt/src/cljfmt/main.clj

```diff
@@ -154,71 +154,91 @@
       (read-string contents)
       (edn/read-string contents))))
 
-(def default-options
-  (merge cljfmt/default-options
-         {:project-root "."
-          :file-pattern #"\.clj[csx]?$"
-          :ansi?        true
-          :parallel?    false}))
-
-(defn merge-default-options [options]
-  (-> (merge default-options options)
-      (assoc :indents (merge (:indents default-options)
+(defn- find-config-file [^String filename]
+  (loop [dir (.getAbsoluteFile (java.io.File. ""))]
+    (when dir
+      (let [file (java.io.File. dir filename)]
+        (if (.exists file)
+          (.getAbsolutePath file)
+          (recur (.getParentFile dir)))))))
+
+(defn- load-configuration []
+  (when-let [config-file (find-config-file ".cljfmt.edn")]
+    {:config-file config-file
+     :config-data (cli-file-reader config-file)}))
```

Comment:
I don't think it's necessary to keep track of the file in this way. Let's keep it simple and just get the configuration.

Reply:
done